### PR TITLE
Updated README.md for the new strategy of deployment

### PR DESCRIPTION
**Pull request by ktp242**
This PR updates README.md for the new strategy for deployment.

### CODE REVIEW

**Comment by holingpoon** [comment]
@ktp242 I may steal this and cross copy-paste into other Travis CI repos. Is that okay with you? Thanks!

**Comment by ktp242** [comment]
@holingpoon Feel free to use it and modify it! :-D
Don't forget to change the branch names for different apps!!

File: README.md [hunk 1]
```diff
@@ -34,6 +34,26 @@ $ npm run dist // Builds dist path & files
 $ APP_ENV=production NODE_ENV=production npm start // Starts localhost:3001 with set APP_ENV
 ```
 
+### Deployment
+NYPL Digital uses Travis to help deploy this application. WIth the configurations in .travis.yml, we can deploy the branch we want by just pushing it and then Travis will take care of the deployment.
```
Comment (holingpoon) [inline README.md]: {{With}} instead of {{WIth}}

File: README.md [hunk 2]
```diff
@@ -34,6 +34,26 @@ $ npm run dist // Builds dist path & files
...
+
+Now we have three environments, `nypl-global-search-production`, `nypl-global-search-qa`, and `nypl-global-search-development`. They are deployed from three branches respectively, `master`, `qa`, and `development`. Once the feature branch is merged into one of these three branches and then pushed, Travis will start to run installation and execute tests. If the tests are passed, Travis will start to deploy to AWS.
+
+We can track the activities of Travis here,
+https://travis-ci.org/NYPL/dgx-global-search
+
+`nypl-global-search-production` and `nypl-global-search-qa` are belonged to `nypl-digital-dev` on AWS. and `nypl-global-search-development` is belonged to `nypl-sandbox`. For checking the activities either of the AWS accounts, log in
```
Comment (EdwinGuzman) [inline README.md]: No need for `are` here.

File: README.md [hunk 3]
```diff
@@ -34,6 +34,26 @@ $ npm run dist // Builds dist path & files
...
+https://console.aws.amazon.com
+
+You will need the credentials. Please contact NYPL Digital if you need one.
+
+#### For deploying a feature branch to one of the environments
+Sometimes, we need to test a feature branch at one of the environments, so we need to deploy the branch. For doing that, change the value of `branch` configuration from `master`, `qa`, or `development` to the name of the feature branch. Save, commit, and push it just as the regular deployments.
```
Comment (EdwinGuzman) [inline README.md]: Where do you change the value of `branch`? This doesn't really say...

**Pull request merged**